saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/metadata/OpenSamlMetadataResolver.java

```diff
@@ -121,6 +123,10 @@ public void setEntityDescriptorCustomizer(Consumer<EntityDescriptorParameters> e
 		this.entityDescriptorCustomizer = entityDescriptorCustomizer;
 	}
 
+	public void setUseFormatPrettyPrintXML(boolean useFormatPrettyPrintXML) {
```

Comment:
Would you please simplify this method to be called `setUsePrettyPrint`?

Reply:
yes, done, i also updated the flag's name for consistency, thank you!